bp/src/Bluepill.m

```diff
@@ -182,7 +182,7 @@ - (void)recover {
 
 // Proceed to next test case
 - (void)proceed {
-    if (self.retries == [self.config.errorRetriesCount integerValue]) {
```

Comment:
It might be also valid to raise an exception instead of continuing here, as the program state is invalid.

Reply:
This can be avoided since the retry count is incremented only in 3 places. In this particular case, I think I know the issue and suggested a bug fix to avoid it.
Also, because this is an internal Bluepill error and extra retries might be harmless if contained by `>=` check, logging an ERROR might be a better approach. If the execution is penalized every time this corner case (or some other bug) is hit, then Bluepill might become unusable until the underlying it is fixed, right?

Reply:
@jerrymarino, Hope we are good here with a logged ERROR message.